04_containers.Rmd

```diff
@@ -637,24 +638,29 @@ sample_meta_file_path <- "data/Mapping_file_ADHD_aggregated.csv"
 tree_file_path <- "data/Data_humanization_phylo_aggregation.tre"
 ```  
 
-Now we can load the biom data into a SummarizedExperiment (SE) object.
+Now we can read in the biom file and convert it into a TreeSE object. In addition, we retrieve the rank names from the prefixes of the feature names and then remove them with the `rankFromPrefix` and `removeTaxaPrefixes` optional arguments.
 
 ```{r}
 library(mia)
 
-# Imports the data
-se <- loadFromBiom(biom_file_path)
+# Read in biom file
+biom_obj <- biomformat::read_biom(biom_file_path)
+
+# Convert biom to TreeSE
+tse <- makeTreeSEFromBiom(biom_obj,
+                          rankFromPrefix = TRUE,
+                          removeTaxaPrefixes = TRUE)
```

Comment:
Why don't we use `loadFromBiom` as it is shorter than calling two different functions?